service/controller/v10/version_bundle.go

```diff
@@ -61,6 +36,6 @@ func VersionBundle() versionbundle.Bundle {
 			},
 		},
 		Name:    "azure-operator",
-		Version: "2.5.0",
+		Version: "3.0.0",
```

Comment:
```suggestion
		Version: "2.6.0",
```

Let's not bump too much here - we usually bump this high if there are many changes in the operator. This here is mainly in k8scc IMO.

Reply:
ok it make sense